packs/hello-universe-1.1.2/manifests/hello-universe.yaml

```diff
@@ -0,0 +1,309 @@
+    spec:
+      serviceAccountName: hello-universe-role
+      containers:
+        - name: api
+          image: ghcr.io/spectrocloud/hello-universe-api:1.0.12
+          ports:
+            - containerPort: 3000
+              name: api
+          env:
+            - name: db-password
+              valueFrom:
+                secretKeyRef:
+                  name: db-password
+                  key: db-password
+            - name: DB_HOST
+              value: "postgres.{{ .Values.namespace }}.svc.cluster.local"
+            - name: PORT
+              value: "3000"
+            - name: DB_USER
+              value: "postgres"
+            - name: DB_NAME
+              value: "counter"
+            - name: DB_ENCRYPTION
+              value: "disable"
+            - name: DB_INIT
+              value: "false"
+            - name: AUTHORIZATION
+              value: "true"
+          resources:
+            limits:
+              memory: "500Mi"
+              cpu: "500m"
+            requests:
+              memory: "500Mi"
+              cpu: "500m"
+          livenessProbe:
+            httpGet:
+              path: /api/v1/health
+              port: 3000
+            initialDelaySeconds: 35
+            periodSeconds: 3
+          readinessProbe:
+            httpGet:
+              path: /api/v1/health
+              port: 3000
+            initialDelaySeconds: 30
+            periodSeconds: 3
+---
+apiVersion: v1
+kind: Service
+metadata:
+  name: api
+  namespace: {{ .Values.namespace }}
+spec:
+  selector:
+    app: api
+  ports:
+    - protocol: TCP
+      port: 3000
+      targetPort: 3000
+  type: ClusterIP # If you want to expose the API service, change this value to LoadBalancer. See below for an example.
+  # type: Loadbalancer
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: ui
+  namespace: {{ .Values.namespace }}
+spec:
+  selector:
+    matchLabels:
+      app: ui
+  replicas: {{ .Values.replicas }}
+  template:
+    metadata:
+      labels:
+        app: ui
+    spec:
+      serviceAccountName: hello-universe-role
+      initContainers:
+        - name: init-container
+          image: busybox
+          command:
+            ["/bin/sh", "-c", "echo 'sleeping for 30 seconds' && sleep 30"]
+      containers:
+        - name: ui
+          image: ghcr.io/spectrocloud/hello-universe:1.1.2-proxy
+          imagePullPolicy: Always
+          ports:
+            - containerPort: 8080
+              name: ui
+          env:
+            - name: TOKEN
+              valueFrom:
+                secretKeyRef:
+                  name: auth-token
+                  key: auth-token
+            - name: API_URI
+              # Leave empty and set QUERY_K8S_API to true to leverage a single loadbalancer.
+              # Otherwise, set to the API service's URL and set QUERY_K8S_API to false
+              value: ""
+            - name: SVC_URI
+              value: "api.{{ .Values.namespace }}.svc.cluster.local:3000"
```

Comment:
This port value would change if we expose the ability to change ports.

Reply:
Resolving this as cannot do.